cellxgene_resources/cellxgene_mods.py

```diff
@@ -42,6 +43,20 @@
 full_obs_standards = portal_obs_fields + curator_obs_fields
 
 
+class CxG_API:
+    scc_repo_loc = os.path.expanduser('~/GitClones/CZI/')
+    api_key_file_path = os.path.expanduser('~/Documents/keys/cxg-api-key.txt')
+    sys.path.append(os.path.abspath(scc_repo_loc + 'single-cell-curation/notebooks/curation_api/python/'))
+
+
+    from src.collection import create_collection,create_revision,get_collection,get_collections,update_collection
+    from src.dataset import create_dataset,delete_dataset,get_dataset,get_datasets,upload_datafile_from_link,upload_local_datafile
+    from src.utils.config import set_api_access_config
+
+
+    set_api_access_config(api_key_file_path)
+
+
```

Comment:
This works correctly and is nice for a cleaner namespace. I haven't done an import into a class but it works and makes sense.

How often do you or anyone else need to set an API key for dev or staging? When I first started, I did that quite a bit as I worked on the basics, but haven't done that recently. That might require setting an instance of the class with an argument of 'dev' or 'staging' instead of this current setup with class-wide attributes

Reply:
Actually just ran into the need to upload to dev today and hit a wall. Should definitely enable that easily. I'll make that update.

Reply:
@brian-mott I applied a fix in 23523fa6ed402b444395ad245887a0465c77e677 that allows for defining an environment. It seems to work, but let me know if that's what you had in mind or if you have another idea for how to implement it.

Reply:
Looks like it works for me for both uploading datasets in `upload_local.ipynb` and pulling API information in `curation_revision.ipynb` This works for both dev and prod.

I don't think there's a need for separate instances of `CxG_API` so I think this setup without any 'self` references should be alright.

My only suggestions would be an elif clause for staging

Reply:
Good point. Added 'staging' option in 255f13d5ef814644cb9259f5d8599125f1a1d9f9